src/containers/my-quizzes/view-quiz-container/ViewQuizContainer.tsx

```diff
@@ -12,7 +13,8 @@ const ViewQuizContainer = ({ questions }: QuizContentProps) => {
       <Typography sx={styles.title}>
         Lorem ipsum dolor sit amet consectetur
```

Comment:
Maybe you should move this test to translations?

Reply:
> Maybe you should move this test to translations?

the same as above. Is it possible to translate it ?